src/pages/Transaction/Tabs/Components/TransactionFunction.tsx

```diff
@@ -4,8 +4,7 @@ import {Types} from "aptos";
 import CurrencyExchangeOutlinedIcon from "@mui/icons-material/CurrencyExchangeOutlined";
 import DescriptionOutlinedIcon from "@mui/icons-material/DescriptionOutlined";
 import {CodeLineBox} from "../../../../components/CodeLineBox";
-
-const BLOCK_MODULE_NAME = "candy_machine_v2";
```

Comment:
wasn't really sure what this BLOCK_MODULE_NAME special code for candy_machine_v2 is for and removed it. Please let me know if this needs to be added back and what this is for.

Reply:
Could you add it back? I'm following up in the slack.